ShareX.UploadersLib/Forms/UploadersConfigForm.resx

```diff
@@ -13483,4 +13483,415 @@ Using an encrypted library disables sharing.</value>
+    <value>NoControl</value>
+  </data>
+  <data name="cbUpleaInstantDownloadEnabled.Location" type="System.Drawing.Point, System.Drawing">
+    <value>102, 68</value>
+  </data>
+  <data name="cbUpleaInstantDownloadEnabled.Size" type="System.Drawing.Size, System.Drawing">
+    <value>151, 17</value>
+  </data>
+  <data name="cbUpleaInstantDownloadEnabled.TabIndex" type="System.Int32, mscorlib">
+    <value>13</value>
+  </data>
+  <data name="cbUpleaInstantDownloadEnabled.Text" xml:space="preserve">
+    <value>Instant Download Enabled</value>
+  </data>
+  <data name="cbUpleaIsPremium.AutoSize" type="System.Boolean, mscorlib">
+    <value>True</value>
+  </data>
+  <data name="cbUpleaIsPremium.Enabled" type="System.Boolean, mscorlib">
+    <value>False</value>
+  </data>
+  <data name="cbUpleaIsPremium.ImeMode" type="System.Windows.Forms.ImeMode, System.Windows.Forms">
+    <value>NoControl</value>
+  </data>
+  <data name="cbUpleaIsPremium.Location" type="System.Drawing.Point, System.Drawing">
+    <value>102, 45</value>
+  </data>
+  <data name="cbUpleaIsPremium.Size" type="System.Drawing.Size, System.Drawing">
+    <value>118, 17</value>
+  </data>
+  <data name="cbUpleaIsPremium.TabIndex" type="System.Int32, mscorlib">
+    <value>12</value>
+  </data>
+  <data name="cbUpleaIsPremium.Text" xml:space="preserve">
+    <value>Is Premium Member</value>
+  </data>
+  <data name="gbUpleaLoginCredentials.Location" type="System.Drawing.Point, System.Drawing">
+    <value>16, 14</value>
+  </data>
+  <data name="gbUpleaLoginCredentials.Size" type="System.Drawing.Size, System.Drawing">
+    <value>428, 145</value>
+  </data>
+  <data name="gbUpleaLoginCredentials.TabIndex" type="System.Int32, mscorlib">
+    <value>12</value>
+  </data>
+  <data name="gbUpleaLoginCredentials.Text" xml:space="preserve">
+    <value>Login Credentials</value>
+  </data>
+  <data name="gbUpleaUserInformation.Location" type="System.Drawing.Point, System.Drawing">
+    <value>450, 14</value>
+  </data>
+  <data name="gbUpleaUserInformation.Size" type="System.Drawing.Size, System.Drawing">
+    <value>350, 107</value>
+  </data>
+  <data name="gbUpleaUserInformation.TabIndex" type="System.Int32, mscorlib">
+    <value>11</value>
+  </data>
+  <data name="gbUpleaUserInformation.Text" xml:space="preserve">
+    <value>User Information</value>
+  </data>
+  <data name="label4.AutoSize" type="System.Boolean, mscorlib">
+    <value>True</value>
+  </data>
+  <data name="label4.ImeMode" type="System.Windows.Forms.ImeMode, System.Windows.Forms">
+    <value>NoControl</value>
+  </data>
+  <data name="label4.Location" type="System.Drawing.Point, System.Drawing">
+    <value>17, 22</value>
+  </data>
+  <data name="label4.Size" type="System.Drawing.Size, System.Drawing">
+    <value>79, 13</value>
+  </data>
+  <data name="label4.TabIndex" type="System.Int32, mscorlib">
+    <value>14</value>
+  </data>
+  <data name="label4.Text" xml:space="preserve">
```

Comment:
Thanks for pointing that out @Jaex. That was actually the original 10 entries for the control left in by VS after the rename in designer. The new entries for lblUpleaEmailAddress were added to the same file, but I hadn't expected that it would not remove the references (or update) those for the old control name. I've just removed all references to label4 (the old control name) in the resx and checked it in. I even did a solution wide search just now to be doubly sure and the only references to label4 are in MovableShowColorForm.Designer.cs but that should be unrelated.